src/erlfmt.erl

```diff
@@ -292,28 +292,34 @@ node_string(Cont) ->
     {String, Anno} = erlfmt_scan:last_node_string(Cont),
     {raw_string, Anno, string:trim(String, both, "\n")}.
 
-format_nodes([{attribute, _, {atom, _, spec}, _} = Attr, {function, _, _} = Fun | Rest], PageWidth) ->
+format_nodes([], _PageWidth) ->
+    [];
+format_nodes(Nodes, PageWidth) ->
+    [$\n | Formatted] = format_nodes_helper(Nodes, PageWidth),
+    Formatted.
+
+format_nodes_helper([{attribute, _, {atom, _, spec}, _} = Attr, {function, _, _} = Fun | Rest], PageWidth) ->
```

Comment:
How about `format_nodes_loop` for the name here?

Reply:
Can do 👍